util/get-net.js

```diff
@@ -1,19 +1,26 @@
 var get = require('lodash.get')
 
 module.exports = function getNet (config) {
-  var connection = get(config, 'connections.incoming.net', [])
-    .find(function (transport) {
-      return isAccessible(transport)
-    })
+  const conns = get(config, 'connections.incoming.net', [])
 
-  if (!connection) return
+  return (
+    conns.find(isPublic) ||
+    conns.find(isLocal) ||
+    conns.find(isDevice)
```

Comment:
THIS is the main change which helps with an offline thing.
Basically I've changed the logic so it tries to find a thing with a transport host/port which is public, then if not that local, then finally falls back to device.